config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -42,6 +42,22 @@ spec:
                 to indicate what labels, annotations, name prefix, etc., should be
                 used when creating the Node.
               type: object
+            providerID:
+              description: ProviderID is the identification ID of the machine provided
+                by the provider. This field must match the provider ID as seen on
+                the node object corresponding to this machine. This field is required
+                by higher level consumers of cluster-api. Example use case is cluster
+                autoscaler with cluster-api as provider. Clean-up login in the autoscaler
+                compares machines v/s nodes to find out machines at provider which
```

Comment:
v/s?

Reply:
Not sure, this was updated in another PR.